collect-adb.rst

```diff
@@ -0,0 +1,115 @@
+*****************************************
+Using Android Debug Bridge with Collect
+*****************************************
+
+This document focuses specifically on using **adb** as a command line tool to perform tasks related to Collect. 
+
+.. _what-is-adb:
+
+What is Android Debug Bridge (adb)?
+====================================
+
+Android Debug Bridge or `adb <https://developer.android.com/studio/command-line/adb.html>`_ is a command line tool which acts as a bridge between the Android device and the terminal. It can control device over USB from a computer, copy files back and forth, install and uninstall apps, run shell commands, and more. To install **adb**, please follow the instructions given `here <https://android.gadgethacks.com/how-to/android-basics-install-adb-fastboot-mac-linux-windows-0164225/>`_. For the developers and users of ODK Collect, the most common uses are:
+
+- include pushing blank forms to SD Card
+- pulling the form database, deleting forms
+- making the **.apk** file from the source code
+
+.. _using-adb-with-collect:
+
+Using adb with Collect
+=======================
+
+Forms can be manipulated from the command line itself. The following sections describe how **adb** can be used to work with the app.
+
+.. _loading-blank-forms-with-adb:
+
+Loading blank forms
+~~~~~~~~~~~~~~~~~~~~
+
+The forms are stored in :guilabel:`sdcard/odk/forms/` folder on the device. They can be downloaded from external source and loaded via a USB device using:
```

Comment:
Use `file` text role for files and directories.

```
:file:`sdcard/odk/forms`

Reply:
> downloaded from external source 

**OR**, you might have built them locally. So it is best to avoid related-but-unneeded editorializing.

> The forms are stored in `sdcard/odk/forms/` folder on the device. They can be downloaded from external source and loaded via a USB device using: (24 words)

Possible correction:


> Forms are stored on the device at `sdcard/odk/forms`. Use `adb push` to load them. (14 words)